.github/workflows/pr-test.yml

```
@@ -39,32 +39,12 @@ jobs:
         run: |
           yarn --frozen-lockfile
 
-      - name: Config git core.quotePath
+      - name: Get changed documents
         run: |
-          # If you don't do this, the get-diff-action won't be able to
-          # notice files that contain non-ascii characters.
-          # I.e.
-          #
-          #    ▶ git status
-          #    Changes not staged for commit:
-          #    ...
-          #   modified:   "files/en-us/glossary/b\303\251zier_curve/index.html"
-          #
-          # But after you set `core.quotePath` you get:
-          #
-          #    ▶ git status
-          #    Changes not staged for commit:
-          #    ...
-          #   modified:   "files/en-us/glossary/bézier_curve/index.html"
-          #
-          # Now, this gets used by the `git diff ...` inside get-diff-action.
-          git config --global core.quotePath false
-
-      - uses: technote-space/get-diff-action@v6.1.0
-        id: git_diff_content
-        with:
-          PATTERNS: files/**/*.+(html|md)
-          SET_ENV_NAME: GIT_DIFF_CONTENT
+          DIFF_DOCUMENTS=$(gh pr view ${{ github.event.number }} --json files --jq ".files.[].path" | egrep -i "^files/.*/.*\.(html|md)$" | tr "\n" " ")
```

Do we really need to specify the PR number, given that GitHub CLI is run in the context of the PR?

```suggestion
          DIFF_DOCUMENTS=$(gh pr view --json files --jq ".files.[].path" | egrep -i "^files/.*/.*\.(html|md)$" | tr "\n" " ")
```

---

seems that we should specific the pr number:

https://github.com/mdn/translated-content/actions/runs/3289036253/jobs/5420089835#step:6:10

---

TIL, thanks for checking! 🙂